package.json

```diff
@@ -24,7 +24,7 @@
     "change": "beachball change",
     "check-for-changed-files": "cd scripts && yarn fluentui-scripts checkForModifiedFiles",
     "checkchange": "beachball check --changehint \"Run 'yarn change' to generate a change file\"",
-    "depcheck": "rnx-dep-check && lage depcheck",
+    "depcheck": "lage depcheck",
```

Comment:
This seems like a regresssion?

Reply:
ugh I keep doing this sorry. I have to delete it in order to get depcheck working locally.

Reply:
Curious, what breaks locally? We could also update our lage config / scripts to just... call rnx-dep-check separately

Reply:
It fails partway and tells us we should be using different versions of react things. Which is fine, but it stops the rest of the command from running when running locally so it doesn't get to the lage depcheck part.